syntaxes/menhir.json

```diff
@@ -66,9 +66,21 @@
           ]
         },
         {
-          "comment": "builtin declaration",
+          "comment": "parameter declaration",
           "name": "keyword.other.menhir",
-          "match": "%(?:parameter|attribute)\\b"
+          "match": "%parameter\\b"
+        },
+        {
+          "comment": "attribute declaration",
+          "begin": "%(?:attribute\\b)?",
```

Comment:
This will break stuff

Reply:
Please be more descriptive.

Can you give an example?

Reply:
This might eat up `%%` when matched, and in general the `?` seems to be unnecessary.